spec/acceptance/basic_spec.rb

```diff
@@ -2,14 +2,14 @@
 
 describe 'Simple installation' do
   before(:context) do
-    if fact('osfamily') == 'RedHat'
```

Comment:
Rather than doing it like this, I'd prefer to handle it centrally: https://github.com/theforeman/puppet-dhcp/pull/173